nhrpd/netlink_arp.c

```diff
@@ -93,20 +97,33 @@ static void netlink_neigh_msg(struct nlmsghdr *msg, struct zbuf *zb)
 	if (!c)
 		return;
 
-	if (msg->nlmsg_type == RTM_GETNEIGH) {
-		debugf(NHRP_DEBUG_KERNEL, "Netlink: who-has %s dev %s",
-		       sockunion2str(&addr, buf, sizeof buf), ifp->name);
+	debugf(NHRP_DEBUG_KERNEL,
+	       "Netlink: %s %s dev %s lladdr %s nud 0x%x "
+	       "cache used %u type %u",
+	       (msg->nlmsg_type == RTM_GETNEIGH)
+		       ? "who-has"
+		       : (msg->nlmsg_type == RTM_NEWNEIGH) ? "new-nei"
+							   : "del-nei",
+	       sockunion2str(&addr, buf[0], sizeof(buf[0])), ifp->name,
```

Comment:
If you dont mind, neigh is used in frr. I dont see any occurences of new-nei.
'new-neigh'

Reply:
ok for new-neigh